velox/functions/prestosql/fuzzer/AggregationFuzzerTest.cpp

```diff
@@ -55,6 +55,13 @@ DEFINE_string(
     "source of truth. Otherwise, use DuckDB. Example: "
     "--presto_url=http://127.0.0.1:8080");
 
+DEFINE_uint32(
```

Comment:
Thanks, can you add this flag to our documentation too ?

Reply:
Thanks for the feedback, added documentation in section `How to integrate`. I am not sure if this is the right place since this argument is for `velox_window_fuzzer_test` and the docs do not delve into fuzzer test arguments. Please let me know if it should be moved elsewhere.

Reply:
Its the right page for sure, but I think its better added to the How to run section.